### DISCUSSION TITLE
using same passkey at different terminals in a store

### DISCUSSION THREAD

**Discussion by rudra610** [teamhanko/hanko#1396] [Q&A]
Hi, is there a feature to use the same passkey at different terminals in a retail store environment setup. and how can the passkeys be synced to these different terminals.

**Comment by FlxMgdnz** [comment]
Passkeys are designed to be personal. Technically, you could set up multiple devices with the same iCloud, Google, or e.g. 1Password account so that passkeys will be synced, but I doubt that's a good idea.
